### DISCUSSION TITLE
Welcome to WHMCS Admin Dark Mode Discussions!

### DISCUSSION THREAD

**Discussion by WevrLabs** [WevrLabs-Group/WHMCS-Blend-Admin-Theme-Dark-Mode#15] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mrkielb** [comment]
Hi
Will you be updating the theme for WHMCS 8.10.x or are there no changes necessary?
Great job btw.

Kind Regards
Martin
